TINY-8632: Fix double separator between Menubar and Toolbar (#7914)
tinymce/tinymce: modules/tinymce/CHANGELOG.md

## Unreleased

### Changed

> - `@toolbar-separator-color` variable is now exclusively used to determine the color of separator between Menubar and Toolbar instead of combination of `@menubar-row-separator-color` and `@toolbar-separator-color` #TINY-8632

Reviewer: This doesn't look accurate, as the `@menubar-row-separator-color` is still used it's just transparent? Additionally the root change more so looks to be that there is no separator between the menubar and toolbar anymore.

Author: Not quite. `@menubar-row-separator-color` is now only applies to the divider between menubar rows (in case there is any menu items wrapping). The separator between menubar and toolbar is gone since v6 (visually) but can be added using `@toolbar-separator-color` variable. I tried to reword changelog entry to make it more clear, hope it helps

- The formatter can now apply a format to a `contenteditable="false"` element by wrapping it. Configurable using the `format_noneditable_selector` option. #TINY-8905
- The autocompleter now supports a multiple character trigger using the new `trigger` configuration #TINY-8887

> ### Changed
> - `@menubar-row-separator-color` variable now has no effect on the divider between Menubar and Toolbar and only controls the color of the separator lines drawn in multiline Menubar. `@toolbar-separator-color` variable is now exclusively used affects the color of separator between Menubar and Toolbar #TINY-8632

Reviewer: ```suggestion
### Changed
- The `@menubar-row-separator-color` variable no longer affects the divider between the Menubar and Toolbar and only controls the color of the separator lines drawn in multiline Menubars. #TINY-8632
- The `@toolbar-separator-color` variable now exclusively affects the color of the separator between the Menubar and Toolbar. #TINY-8632
```
This is 2 separate things and is too long for a changelog anyways, so splitting it makes sense to me. This also still didn't really make sense so I've tried to reword it a little.

Reviewer: @noxuhax why was this marked as resolved without making any changes or leaving any comments?

Author: @lnewson my bad, I did not get notifications about new comments so I was convinced that I’m marking an old thread as resolved. I agree, splitting this entry into two makes more sense.

> - `@menubar-row-separator-color` variable now has no effect on the divider between Menubar and Toolbar and only controls the color of the separator lines drawn in multiline Menubar. `@toolbar-separator-color` variable is now exclusively used affects the color of separator between Menubar and Toolbar #TINY-8632

Reviewer 2: Should this some how explain that the variable here is a skin variable? Also does it effect lines drawn in a multiline toolbar? I guess this log line only makes sense to people who know that these things are anyway. Maybe we could stick details into the release notes.

Author: > Also does it effect lines drawn in a multiline toolbar?

No changes here, it is still controlled via `@toolbar-row-separator-color`.

I believe [changes suggested by Lee](https://github.com/tinymce/tinymce/pull/7914/commits/2d9b6200e7fef3dcad2f1d66777d2c1373572cdf) address your concerns about the length and make the fact that variables are skin-related more clear.

Reviewer 2: Yes, perfect.
